flask_babel/__init__.py

```diff
@@ -214,7 +214,12 @@ def get_translations():
     ctx = _get_current_context()
 
     if ctx is None:
-        return support.NullTranslations()
+        try:
+            return support.NullTranslations()
+        except AttributeError:
+            # Some versions of babel have NullTranslations() in gettext instead of support module
+            import gettext
+            return gettext.NullTranslations()
```

Comment:
Hmm... I believe support.NullTranslations provides more methods and attributes than gettext.NullTranslations. I'm not sure if it is wise for get_translations() to return two classes with different callables